<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDataFile</class>
 <widget class="QWidget" name="ChangeDataFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Data File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QComboBox" name="comboMode">
     <item>
      <property name="text">
       <string>writing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>appending</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushBrowse">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineFileLocation"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>New Data File: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonDialog">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
